.github/workflows/codeql-analysis.yml

```diff
@@ -56,6 +56,7 @@ jobs:
     - if: matrix.language == 'java'
       name: Build Android
       run: |
+        export NDK_HOME=$ANDROID_NDK_LATEST_HOME
```

Comment:
This is an environment variable in the runner, right?

Reply:
Yes, `ANDROID_NDK_LATEST_HOME` is the env variable exposed by GH. Idk, why they didn't use something like `NDK_HOME` or `ANDROID_NDK_HOME` which is the default, in that case this re-export would not be needed...